.pullapprove.yml

```
@@ -534,8 +534,40 @@ groups:
         - ~framework-global-approvers
         - ~framework-global-approvers-for-docs-only-changes
 
+  # =========================================================
+  #  Framework: Benchmarks
+  # =========================================================
+  fw-benchmarks:
+    conditions:
+      - >
+        contains_any_globs(files, [
+          'modules/benchmarks_external/**',
+          'modules/benchmarks/**'
+          ])
+    reviewers:
+      users:
+        - IgorMinar
+        - kara
+        - pkozlowski-opensource
+      teams:
+        - ~framework-global-approvers
 
   # =========================================================
+  #  Framework: Playground
+  # =========================================================
+  fw-playground:
+    conditions:
+      - >
+        contains_any_globs(files, [
+          'modules/playground/**',
```

```suggestion
          'modules/playground/**'
```

(fonr consistency :grin:)